### PR TITLE
💬 Change "fake results" to "zeroes"

### PR DESCRIPTION
**Pull request by flowirtz**

### CODE REVIEW

**Review by peterdudfield — APPROVED** [review]

File: src/main.py [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 version = "0.1.5"
 description = """
-The Nowcasting API is still under development. It only returns fake results.
+The Nowcasting API is still under development. It only returns zeroes for now.
```
Comment (peterdudfield) [inline src/main.py:15]: or is it zeros?

Comment (peterdudfield) [reply src/main.py:15]: https://english.stackexchange.com/questions/3824/what-is-the-plural-form-of-zero

Comment (flowirtz) [reply src/main.py:15]: apparently both is correct haha: https://en.wiktionary.org/wiki/zero#Noun

Complicated language. I don't mind, whichever one u prefer! 😊

Comment (peterdudfield) [reply src/main.py:15]: It's interesting to note that the Oxford English Dictionary's Oxford Dictionaries site's sole definition of zeroes is related to zero as a verb, e.g. "watch as he zeroes his sights on the target"; not as the plural of zero. It states that the plural of zero is zeros.

Comment (peterdudfield) [reply src/main.py:15]: lets go zeros please - now i have looked at it too long

Comment (flowirtz) [reply src/main.py:15]: hahahahhahaha sure

Comment (flowirtz) [inline src/main.py:15]:
```suggestion
The Nowcasting API is still under development. It only returns zeros for now.
```

**Pull request merged**